src/components/navigation/top-navigation.vue

```diff
@@ -45,6 +45,9 @@ export default {
     },
     imgSrc (img) {
       return require('~/assets/logos/' + img)
+    },
+    expandDhoList () {
+      if (this.dhos.length) { this.expanded = !this.expanded }
```

Comment:
@evgenibir This will fail if `this.dhos` is null or undefined

Reply:
@Alex-IS fixed